profile/src/main/java/com/fightpandemics/profile/ui/profile/ProfileSignedOutFragment.kt

```diff
@@ -0,0 +1,55 @@
+package com.fightpandemics.profile.ui.profile
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.fightpandemics.core.utils.setBooleanPreference
+import com.fightpandemics.profile.R
+import kotlinx.android.synthetic.main.profile_signed_out_fragment.*
+import kotlinx.android.synthetic.main.profile_toolbar.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+
+class ProfileSignedOutFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.profile_signed_out_fragment, container, false)
+    }
+
+    @ExperimentalCoroutinesApi
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        bindListeners()
+    }
+
+    @ExperimentalCoroutinesApi
+    private fun bindListeners() {
+        overview.visibility = View.INVISIBLE
+        toolbar.setOnMenuItemClickListener {
+            when (it.itemId) {
+                R.id.settings -> {
+                    findNavController().navigate(com.fightpandemics.R.id.action_profileSignedOutFragment_to_indivProfileSettings)
+                    true
+                }
+                else -> {
+                    super.onOptionsItemSelected(it)
+                }
+            }
+        }
+
+        logInBtnWhite.setOnClickListener {
+            findNavController().navigate(com.fightpandemics.R.id.action_profileSignedOutFragment_to_signInFragment)
+            setBooleanPreference(activity?.applicationContext, "isUserSignedInFromProfile", true)
```

Comment:
Mayorjay I think is not good to call preferences from fragment becaus this bring business logic to fragment class. You should use a view module to do that. Also the shared preferences should not be directly setted from ViewModel we should create all data structure to do that because in some point we could change this to a webserver and you only need to refactor the data layer and not all others layers. 

Take at AuthTokenLocalDataSourceImpl, LoginRepositoryImpl. The AuthTokenLocalDataSourceImpl is using preference storage as well. You can use this class to have a method like isLoggedIN. Refactoring this I think it will be ok to approve the PR